### PR TITLE
Domination round 1

### PR DESCRIPTION
**Pull request by ericwm76**
This creates the timer for the Domination round and instantiates new instances of DominationTurn and Round when you get past Round 2. The code isn't super pretty - needs some refactoring to make sure that every method lives where it should, get rid of overly long functions and nested ifs - but it's working. I will continue to work on getting the scores and points to update later this evening.

### CODE REVIEW

File: src/DominationTurn.js [hunk 1]
```diff
@@ -36,7 +36,40 @@ class DominationTurn extends Turn {
...
+
+  checkGuess(guess) {
+    return this.currentRound.answerStrings.includes(this.capitalizeGuess(guess));
+  }
+
+  spliceAnswers(guess) {
+    let i = this.currentRound.answerStrings.findIndex(answer => answer === this.capitalizeGuess(guess))
+    this.currentRound.answerStrings[i] = 'false';
+    this.currentRound.endRound();
+  }
```
Comment (edwindelbosque) [inline src/DominationTurn.js]: Are we using this splicing method again? or is it creeping from past commits?

File: src/index.js [hunk 2]
```diff
@@ -106,9 +111,44 @@ function displayStartButton() {
...
+const startTimer = () => {
+  const counter = setInterval(countdown, 1000);
+
+  function countdown() {
+    timeLeft -= 1;
+
+    if (timeLeft < 0) {
+       clearInterval(counter);
+       dominationRound.endDominationRound();
+       return;
```
Comment (edwindelbosque) [inline src/index.js]: Interesting method, I like it.
I was thinking something very similar in my head.

File: src/DominationTurn.js (hunk 1 above)
Comment (ericwm76) [reply src/DominationTurn.js]: I just copied some of the methods over from the RegularTurn into DominationTurn, just to have something to mess with. Once I get it working I'll change them.

**Comment by ZoeKHarvey** [comment]
Does this pr have any bugs with functionality? Trying to debug my stuff but wanted to work with the most recent version

**Pull request merged**